### ISSUE TITLE
libRNSnackbar.a was built for newer iOS version (9.1) than being linked (8.0)

### ISSUE THREAD

**Issue by dantman** [cooperka/react-native-snackbar#48]
This warning is showing up:
```
object file (/Users/daniel/Library/Developer/Xcode/DerivedData/MyApp-eukdcxeysdexiigripbtdscwtxsa/Build/Products/Debug-iphonesimulator/libRNSnackbar.a(RNSnackBarView.o)) was built for newer iOS version (9.1) than being linked (8.0)
```

It looks like the podspec's target is 8.0 but the xcode project's settings are 9.1.

**Comment by cooperka** [comment]
Many RN projects still target a minimum of iOS 8.0, so I think I'll keep it this way. The warning seems to just be letting you know the minimum versions are inconsistent. I assume if you want to, you can link using `react-native link` instead of cocoapods to avoid this warning.

I'll close this, but thanks for the report. Let me know if this warning ends up causing any issues for you.

**Comment by dantman** [comment]
@cooperka No, **my** project's target is `8.0`. This warning is happening because **react-native-snackbar**'s target is `9.1`. I **am** using `react-native link`. I just made a note that react-native-snackbar's cocoapods target is `8.0` so the cocoapods users won't see this warning, but `react-native link` users like me will.

https://github.com/cooperka/react-native-snackbar/blob/master/ios/RNSnackbar.xcodeproj/project.pbxproj#L209

**Comment by cooperka** [comment]
I see, thanks for clarifying. This should definitely be updated. Feel free to submit a PR, otherwise I'll get to it within the next few days.

**Comment by cooperka** [comment]
Published in v0.4.3. Thanks again for the report; this project was inherited from a previous team so some of the configs like this may have been non-standard. Hopefully this is the last one!
